bin/pc2installsandbox

```diff
@@ -26,6 +26,12 @@ PC2_CGROUP_PATH=$CGROUP_PATH/pc2
 
 # ------------------------------------------------------------
 
+euid=$(id -u)
```

Comment:
The script uses the '`sudo`' command on the commands that require root.  This script does not necessarily have to run as root, but, as a user than can become root using `sudo`.  I think adding this restriction changes the operation of the script.

Reply:
I ran this script on my own machine and it failed due to permissions unless it was run as root (at the `echo > ...` commands). I can revert this, but perhaps these commands that require higher permissions should be investigated.

A few notable lines: line 74, and line 120